src/events/ready.ts

```diff
@@ -7,17 +7,21 @@ import {
 } from "discord.js";
 import { ctx } from "../ctx";
 import { logger } from "../logger";
+import { BirthdayMessage } from "../util/birthdayCommand/loop";
 
 export default async function ready(client: Client) {
 	logger.info(`Successfully logged in as ${client.user?.username}.`);
 	logger.info(`Watching over ${client.guilds.cache.size} guilds.`);
 
-	logger.info("Publishing commands...");
-	await updateRegisteredCommands(client).then(() => logger.info("Finished publishing commands."));
-
 	logger.info("Starting reminder scheduler...");
 	ReminderCommand.startScheduler(client);
 
+	logger.info("Starting Birthday scheduler...");
+	BirthdayMessage(client);
```

Comment:
Could we keep this consistent with the reminder and have a startScheduler method on the command?